### PR TITLE
[Security] Fix security warning; crazy node-gyp console error

### PR DESCRIPTION
**Pull request by damassi**
Fixes Github security error and the crazy node-gyp c++ output when running yarn install

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/babel-plugin-relay-1.5.0-artsy.5.tgz",
     "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-compiler-1.5.0-artsy.5.tgz",
     "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-runtime-1.5.0-artsy.5.tgz",
-    "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/react-relay-1.5.0-artsy.5.tgz"
+    "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/react-relay-1.5.0-artsy.5.tgz",
+    "fsevents": "~1.2.4"
```
Comment (damassi) [inline package.json]: For future ref, fs events above this version fixes that terminal output. Had to use yarn resolutions to enforce it.

**Comment by damassi** [comment]
ugh, not sure whats up with this circle error

**Comment by eessex** [comment]
This circle error is definitely a new one to me -- ok to close or do you want to keep debugging?

**Comment by damassi** [comment]
Ahh yeah, I'll give this a look this week

**Comment by eessex** [comment]
Resolved in #2738

**Pull request closed without merging**